src/SFA.DAS.AssessorService.Web/Controllers/CertificateOptionController.cs

```diff
@@ -54,7 +54,7 @@ private async Task<IActionResult> LoadViewModel(string view, bool? isFromStandar
             else
                 SessionService.Remove("redirecttocheck");
                 
-            var sessionString = SessionService.Get("CertificateSession");
+            var sessionString = SessionService.Get(nameof(CertificateSession));
```

Comment:
There is very similar code to this in lots of places perhaps a generic method on the base class, the different seems to be the name of the view model

Reply:
And this was supposed to be about checking for the empty session - I don't know why the comments are not on the correct lines

Reply:
Have also refactored into base class 👍